Examples/Io/Root/src/RootAthenaDumpReader.cpp

```diff
@@ -265,11 +340,46 @@ ActsExamples::ProcessCode ActsExamples::RootAthenaDumpReader::read(
                                     activation);
     }
 
-    ACTS_VERBOSE("Cluster " << im << ": " << cluster.channels.size()
-                            << "cells, dimensions: " << cluster.sizeLoc0 << ", "
-                            << cluster.sizeLoc1);
+    cluster.globalPosition = {CLx[im], CLy[im], CLz[im]};
+
+    ACTS_VERBOSE("CL shape: " << cluster.channels.size()
+                              << "cells, dimensions: " << cluster.sizeLoc0
+                              << ", " << cluster.sizeLoc1);
 
     clusters[im] = cluster;
+
+    // Measurement creation
+    ACTS_VERBOSE("CL loc dims:" << CLloc_direction1[im] << ", "
+                                << CLloc_direction2[im] << ", "
+                                << CLloc_direction3[im]);
+    const auto& locCov = CLlocal_cov->at(im);
+
+    DigitizedParameters digiPars;
+    if (type == ePixel) {
+      digiPars.indices = {Acts::eBoundLoc0, Acts::eBoundLoc1};
+      digiPars.values = {CLloc_direction1[im], CLloc_direction2[im]};
+      assert(locCov.size() == 4);
+      digiPars.variances = {locCov[0], locCov[3]};
+    } else {
+      // TODO is this correct ???
+      digiPars.values = {CLloc_direction2[im]};
+      digiPars.indices = {Acts::eBoundLoc1};
```

Comment:
is it? 😄

Reply:
Ah yeah good that you mention, I should check before merging^^